doc/api/deprecations.md

```diff
@@ -2623,14 +2623,14 @@ async function openAndClose() {
 changes:
   - version: REPLACEME
     pr-url: https://github.com/nodejs/node/pull/32217
-    description: Runtime deprecation.
+    description: Documentation-only deprecation.
 -->
 
-Type: Runtime
+Type: Documentation-only (supports [`--pending-deprecation`][])
 
-A CJS module can access the first module that required it using `module.parent`.
-This feature does not work when a module is imported using ECMAScript modules
-specification, therefore it is deprecated.
+A CommonJS module can access the first module that required it using
+`module.parent`. This feature does not work when a module is imported using
+ECMAScript modules specification, therefore it is deprecated.
```

Comment:
The grammar structure here feels weird, particularly the use of “imported using ECMAScript modules specification”.